template/asyncapi.js

```diff
@@ -0,0 +1,555 @@
+import React from "react";
+import { File, Text } from "@asyncapi/generator-react-sdk";
+import { generateExample, getHeadersExamples, getPayloadExamples } from "@asyncapi/generator-filters";
+
+export default function({ asyncapi, params }) {
+  return (
+    <File name={params.outFilename || 'asyncapi.md'}>
```

Comment:
```suggestion
    <File fileName={params.outFilename || 'asyncapi.md'}>
```

Reply:
Why `fileName` here? We know it's the name of the file because the component is `<File>`. A bit repetitive IMHO.

Reply:
Ah, right, I'll change the props back but letting the metadata structure keep `fileName` 👍